web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -0,0 +1,125 @@
+import classnames from "classnames";
+import React, {
+	ReactElement,
+	ReactNode,
+	isValidElement,
+	cloneElement,
+} from "react";
+
+import styles from "./Testimonial.module.scss";
+
+export interface TestimonialProps {
+	variant?: "default" | "transparent" | "fullWidth" | "fullWidthWhite";
+	children?: ReactNode[] | ReactNode; //TODO does this component need to be able to accept children?
+	className?: string;
+	image?: ReactElement;
+	quoteText: string;
+	quoteName: string;
+	quoteRole: string;
+	headingLevel?: 2 | 3 | 4 | 5 | 6;
+	link?: ReactElement | null;
+}
+
+export interface TestimonialLinkProps {
+	[prop: string]: unknown;
+	destination?: string;
+	elementType?: React.ElementType;
+	method?: string;
+}
+
+export const Testimonial: React.FC<TestimonialProps> = (
+	props
+): JSX.Element | null => {
+	const {
+		variant = "default",
+		quoteText,
+		quoteName,
+		quoteRole,
+		children,
+		className,
+		headingLevel = 4,
+		link = undefined,
+		image,
+		...rest
+	} = props;
+
+	const kebabCaseVariantClassName = variant
+		? `${variant.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase()}`
+		: "";
+
+	const testimonialClasses = classnames(
+		styles.testimonial,
+		{
+			[styles[`testimonial--${kebabCaseVariantClassName}`]]: variant,
+		},
+		className
+	);
+
+	const cloneElementWithClassNames = <T extends ReactElement>(
+		element: T | undefined,
+		...additionalClassNames: string[]
+	): T | null => {
+		if (!isValidElement(element)) return null;
+
+		const { className, alt, ...restProps } = element.props as {
+			className?: string;
+			alt?: string;
+			[key: string]: unknown;
+		};
+
+		return cloneElement(element, {
+			...restProps,
+			className: classnames(className, ...additionalClassNames),
+			...(element.type === "img" && { alt: alt || "Testimonial image" }),
```

Comment:
Do we need the default here (and if so is it working as we'd expect)? In the full width examples where no alt text is provided we are just rendering an empty alt tag